### ISSUE TITLE
Build instructions in README?

### ISSUE THREAD

**Issue by userappgate** [jpakkane/parzip#1]
Hello.

I wanted to try this tool but I have never seen meson in my life so I am not sure how to debug it and it won't start any compilation:

```
The Meson build system
Version: 0.45.1
Source dir: /build/project/parzip/parzip-master
Build dir: /build/project/parzip/parzip-master/build
Build type: native build
Project name: parzip
Native C++ compiler: c++ (gcc 7.5.0 "c++ (Ubuntu 7.5.0-3ubuntu1~18.04) 7.5.0")
Build machine cpu family: x86_64
Build machine cpu: x86_64
Found pkg-config: /usr/bin/pkg-config (0.29.1)
Native dependency zlib found: YES 1.2.11
Native dependency liblzma found: YES 5.2.2
Dependency threads found: YES
Native dependency gtk+-3.0 found: YES 3.22.30


test/meson.build:1:0: ERROR: Include directory to be added is not an include directory object.

A full log can be found at /build/project/parzip/parzip-master/build/meson-logs/meson-log.txt
```

I installed all the dependencies listed in the meson file, so it is saying YES to everything.

**Comment by userappgate** [comment]
It looks like the old meson shipped in ubuntu is the issue. On debian testing with 0.54.3 it works.

**Comment by mflage** [comment]
It would be helpful to others that you actually showed the commands you issued to build this. Because I've hit the same obstacle. I'm not able to build it and I have no idea how to use the meson build system.

**Comment by userappgate** [comment]
Ah sorry I can't remember. It was 2 months ago and since then I noticed that the project only zips multiple files in parallel. But since I have a large file to zip, it was useless to me.

Sorry.

**Comment by mflage** [comment]
For completeness, I did ```meson build``` which set up a new folder called "build" and I then ran ```ninja``` inside that. Quite simple, once you know what to do. I guess this issue can be closed now.

**Comment by userappgate** [comment]
Eh to me the needed libraries weren't downloaded. It'd be nice to have a README specify which dependencies and versions are required.

**Comment by mflage** [comment]
> Eh to me the needed libraries weren't downloaded. It'd be nice to have a README specify which dependencies and versions are required.

Yeah, but I just guessed based on the output from ```meson build```. But yeah, I totally agree with you, the project needs some more documentation on how to build - and use. But as you say, if the "parallism" here really just is that it zips several files in parallel, then it's really of no use - at least to me.

**Comment by userappgate** [comment]
Several files within the same .zip file.

Unfortunately I needed to speed up the zipping of a single huge file :)
